changelog.md

```diff
@@ -1,5 +1,14 @@
 # Revision history for waargonaut
 
+## 0.2.0.3  -- 2018-11-13
```

Comment:
Please make this 0.2.1.0 since there are new non-orphan instances https://pvp.haskell.org/